packages/components/atoms/f-popover/stories/Popover.stories.js

```diff
@@ -1,20 +1,36 @@
-// Uncomment the import below to add prop controls to your Story (and add `withKnobs` to the decorators array)
-// import {
-//     withKnobs, select, boolean
-// } from '@storybook/addon-knobs';
 import { withA11y } from '@storybook/addon-a11y';
-import VPopover from '../src/components/Popover.vue';
+import Popover from '../src/components/Popover.vue';
+import Card from '../../f-card/src/components/Card.vue';
 
 export default {
     title: 'Components/Atoms',
     decorators: [withA11y]
 };
 
-export const VPopoverComponent = () => ({
-    components: { VPopover },
+export const PopoverComponent = () => ({
+    components: { Popover, Card },
     props: {
     },
-    template: `<v-popover />`
+    template: `
+        <div style="position: relative; padding: 16px;">
+            Some text
+            <popover 
+                style="position: absolute;
+                        top: 100%;
+                        width: 300px;
+                        opacity: 1;
+                        z-index: 99999;
+                        left: 0;">
+                <card
```

Comment:
Do you need the card component in this base story? Would be better if it could just be a header/text passed in so that we don't tie the two components together in the base story if possible (can always have additional stories to show other ways the component can work)